features/core-download.feature

```diff
@@ -18,6 +18,12 @@ Feature: Download WordPress
     And I run `cd inner && wp core download`
     Then the inner/wp-settings.php file should exist
 
+    When I try `WP_CLI_STRICT_ARGS_MODE=1 wp core download --path=inner`
```

Comment:
`WP_CLI_STRICT_ARGS_MODE` shouldn't be necessary here. Is there a particular reason you're using it?